pkg/distributor/distributor.go

```diff
@@ -360,6 +360,7 @@ func (d *Distributor) Push(ctx context.Context, req *client.WriteRequest) (*clie
 		return nil, err
 	}
 
+	source := req.Source
```

Comment:
This doesn't seem to be adding much; just use `req.Source` when you need it.

Reply:
I've fixed it.